en/components/expansion_panel.md

```diff
@@ -248,6 +188,125 @@ After applying all of the changes to our initial component, here is the final re
 
 The `IgxExpansionPanel` control allows all sort of content to be added inside of the `igx-expansion-panel-body`. It can render [`IgxGrid`](grid/grid.md)s, [`IgxCombo`](combo.md), charts and even other expansion panels!
 
+## Styling
+The Expansion Panel comes without specific borders or paddings defined for the body content. You can either manually set the Expansion Panel body element's padding or wrap the Expansion Panel body in an `igxCard` . You can refer to the [Weather Forecast Sample](#weather-forecast-sample) for the latter.
+
+### Styling the header
+The [`IgxExpansionPanelComponent`]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html) allows for easy customization of [the header]({environment:angularApiUrl}/classes/igxexpansionpanelheadercomponent.html).
+The default icon for the toggle state of the control can be templated. The position of this icon is also configurable - either the start or end of the header. It can also be hidden.
+
+Configuring the position of the header icon can be done through the [`iconPosition`]({environment:angularApiUrl}/classes/igxexpansionpanelheadercomponent.html#iconposition) input on the `igx-expansion-panel-header`. The posible options for the icon position are **left**, **right** and **none**. The next code sample demonstrates how to configure the component's button to go on the *right* side.
+
+```html
+<!-- in expansion-component.component.html -->
+    <igx-expansion-panel>
+    ...
+        <igx-expansion-panel-header [iconPosition]="'right'">
+        </igx-expansion-panel-header>
+    </igx-expansion-panel>
+```
+> Note: The [`iconPosition`]({environment:angularApiUrl}/classes/igxexpansionpanelheadercomponent.html#iconposition) property works with `RTL` - e.g. an icon set to show up in **right** will show in the leftmost part of the header when RTL is on.
+
+We can also override the default icon that is used in the control by passing content in an `igx-expansion-panel-icon` tag:
+```html
+<!-- in expansion-component.component.html -->
+    <igx-expansion-panel>
+    ...
+        <igx-expansion-panel-header [iconPosition]="'right'">
+        ...
+            <igx-expansion-panel-icon>
+                <span class="example-icon" *ngIf="panel.collapsed">Show more</span>
+                <span class="example-icon" *ngIf="!panel.collapsed">Collapse</span>
+            </igx-expansion-panel-icon>
+        </igx-expansion-panel-header>
+    </igx-expansion-panel>
+```
+```css
+<!-- in expansion-component.component.scss -->
+.example-icon {
+    font-size: 12px;
+    font-weight: 600px;
+}
+```
+Our component will now render "Show More" when the panel is collapsed and "Collapse" once it's fully expanded.
+
+### Import theme
+Using the [Ignite UI for Angular Theming](themes/index.md), we can alter the **igx-expansion-panel** appearance. Now we are going to style a simplified version of the above [animations sample](#using-specific-animation) and modify it to have а custom appearance.
```

Comment:
Remove the part about "above animation sample...". The sample is now _below_ the styling section. Since users are not familiar with it yet, no need to mention it at all.